portfolio/src/main/webapp/comments.js

```diff
@@ -0,0 +1,44 @@
+function displayComments(){
+    fetch('/comments-data')
+    .catch(error => console.log('failed to fetch comments data from server'))
+    .then(response => response.json())
+    .catch(error => console.log('failed to parse comments'))
+    .then(response => addAllCommentsToDOM(response))
+    .catch(error => console.log('failed to print comments to DOM'));
+}
+
+function createCardHolder(){
+    var card_holder = document.createElement("div");
+    card_holder.setAttribute('class', 'card-holder-small');
+    return card_holder;
+}
+
+function createCard(){
+    var card = document.createElement("div");
```

Comment:
Nit:I would probably name this `createCardElement` to try to make it more obvious that it is an html element and not a model class for example. Same below.